module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2029,6 +2056,11 @@ protected function getItemStatusesForBiblio($id, $patron = null)
             return [];
         }
 
+        // Get the total number of items returned from the API call and set it to
+        // a class variable. It is then used in VuFind\RecordTab\HoldingsILS for
+        // the items paginator
+        $results['total'] = (int)$result['data']['items_total'];
+
         $statuses = [];
```

Comment:
This variable is no longer used, right?

Reply:
Yes that's right. I've removed that line in my follow up